cfme/tests/containers/test_containers_projects_summary.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.project import Project
+from cfme.web_ui import CheckboxTable
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+
+projects_properties_fields = ['Name', 'Creation timestamp', 'Resource version']
+projects_relationships_fields = ['Containers Provider', 'Routes', 'Services', 'Replicators',
+                                 'Pods', 'Nodes']
+
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+
+
+# CMP - 9523
+
+
+def test_containers_projects_summary_relationships(provider):
+    """ Relationships fields tests in Project summary
+        This test checks correct population of the Relationships Fields in Containers Projects'
+        details menu
+        Steps:
+            * Goes to Containers -- > Projects menu
+            * Go through each Container Project in the menu and check validity of
+            Relationships fields
+        """
+    sel.force_navigate('containers_projects')
+    project_name = [r.name.text for r in list_tbl.rows()]
+    for name in project_name:
+        for field in projects_relationships_fields:
+            obj = Project(name, provider)
+            val = obj.get_detail('Relationships', field)
+            assert val
```

Comment:
Wait, hold the horses.. The val here might return strings and numbers. Returning a zero here is not necessarily a problem. 
For example, Containers Provider is a string, Routes can definitely bring back 0 (Not a healthy situation but not necessarily a bug)